src/Core/KeyManagement/Models/Response/MasterPasswordUnlockResponseModel.cs

```diff
@@ -6,15 +6,15 @@ namespace Bit.Core.KeyManagement.Models.Response;
 
 public class MasterPasswordUnlockResponseModel
 {
-    public required MasterPasswordUnlockKdfResponseModel Kdf { get; init; }
-    [EncryptedString] public required string MasterKeyEncryptedUserKey { get; init; }
-    [StringLength(256)] public required string Salt { get; init; }
+    [Required] public required MasterPasswordUnlockKdfResponseModel Kdf { get; init; }
```

Comment:
💭  It looks like in newer versions of the swagger NuGet package, the `required` keyword is honored. Good to know we need both for now.